classes/db/DAO.inc.php

```diff
@@ -98,11 +99,15 @@ function retrieveRange($sql, $params = [], $dbResultRange = null, $callHooks = t
 
 	/**
 	 * Count the number of records in the supplied SQL statement (with optional bind parameters parameters)
-	 * @param $sql string SQL query to be counted
+	 * @param $sql string|Builder SQL query to be counted
 	 * @param $params array Optional SQL query bind parameters
```

Comment:
Should note that if a Builder is passed for `$sql`, then `$params` will not be used.

Reply:
Ok!